rfcs/RFC-Upload-From-Dropbox.md

```diff
@@ -0,0 +1,367 @@
+| Link                  | a shared link to the file                                                                                                                             |
+| submission_id         | identifies the submission                                                                                                                             |
+| _Optional_ dataset_id | identifies the dataset being uploaded. Used to check the status of the upload.                                                                        |
+| _Optional_ overwrite  | A boolean, indicating if the user would like to overwrite the file that was uploaded. The upload status must be complete for the dataset_id provided. |
+
+**Response:**
+
+| Key        | Description                                                               |
+| ---------- | ------------------------------------------------------------------------- |
+| dataset_id | identifies the dataset the file will be associated with after validation. |
+| status     | Provides the current status of the upload                                 |
+| message    | An error message if the upload failed                                     |
+
+**Error Responses:**
+
+| Code | Description                                                                                                      |
+| ---- | ---------------------------------------------------------------------------------------------------------------- |
+| 401  | if dataset_id or submission_id does not exist, or if the user does not own the submission or upload in-progress. |
+| 400  | if the file type is invalid                                                                                      |
+
+#### DELETE submission/{submission_id}/upload
+
+Cancels an existing upload job. Any data that has started to upload is removed, and the upload status is changed to
+cancel pending until the job has been cleared up.
+
+**Request:**
+
+| Parameter     | Description                           |
+| ------------- | ------------------------------------- |
+| submission_id | identifies the submission             |
+| dataset_id    | identifies the dataset being canceled |
+
+**Response:**
+
+| Key    | Description                                |
+| ------ | ------------------------------------------ |
+| status | Provides the current status of the upload. |
+
+**Error Responses:**
+
+| Code | Description                                                                                                             |
+| ---- | ----------------------------------------------------------------------------------------------------------------------- |
+| 401  | if dataset_id or submission_id does not exist, or if the user does not own the submission or upload in-progress upload. |
+| 400  | if the parameters supplied are invalid.                                                                                 |
+
```

Comment:
What's the API that one would use to check the progress of the dataset?

Reply:
I added a get upload endpoint.

Reply:
Where is this GET documented?